### PR TITLE
Use scale instead of pixels, fixes #38

### LINKED ISSUE #38: ,
,

### PR DESCRIPTION
**Pull request by rubenstolk**
:eyes:
- [x] Zoom: 100% after and before this PR:
  screen shot 2016-03-29 at 21 57 43
- [x] Zoom: 90% after and before this PR:
  screen shot 2016-03-29 at 21 57 33

### CODE REVIEW

**Comment by rubenstolk** [comment]
Is this something scheduled?

**Comment by rubenstolk** [comment]
It looks like this was already fixed in https://github.com/PolymerElements/paper-radio-button/pull/91. That mostly fixes it although the result is not perfect, still a bit off in certain zoom levels.

I wouldn't mind taking a look at it again, but before I do I need to know if there is a chance of getting a merged, so someone please let me know!

@hotforfeature @bicknellr @notwaldorf 

FYI these are screenshots from 90% and 110% in Chrome with the current master branch, definitely better than before but I would still vote for what I suggested with this PR:

screen shot 2016-06-09 at 19 50 56

screen shot 2016-06-09 at 19 51 06

**Comment by hotforfeature** [comment]
I don't see why not, it may even be easier now. I haven't tried it myself, but if you set `#onRadio` width and height to 100%, then the checked state to `scale(0.5)` using the current master branch, I think that'd accomplish what you were going for.

**Comment by rubenstolk** [comment]
@hotforfeature @bicknellr I've updated this PR:
- BEFORE 90%
  screen shot 2016-06-10 at 10 48 23
- AFTER 90%
  screen shot 2016-06-10 at 10 48 38
- BEFORE 110%
  screen shot 2016-06-10 at 10 48 53
- AFTER 110%
  screen shot 2016-06-10 at 10 49 03

**Comment by bicknellr** [comment]
It's better but I'm still seeing a lot of rendering inconsistency; here's what it looks like 90%:
screen shot 2016-06-10 at 14 09 21
screen shot 2016-06-10 at 14 08 10
screen shot 2016-06-10 at 14 10 00
To really squash this bug, I think we need to switch to SVG from divs. All of this seems to result from each browser's individual behavior for aligning boxes with device pixels to prevent border anti-aliasing.

**Comment by rubenstolk** [comment]
@bicknellr that's true but at least now we have the best situation that can be achieved with HTML/CSS. I agree that `svg` is better but with this PR in I wouldn't consider it as a bug anymore as it just behaves like anything else on zoom levels. We would be as good as something like https://material.angularjs.org/latest/demo/radioButton now.

**Comment by bicknellr** [comment]
I think that's a reasonable argument. I filed #96 about converting to SVG, but, like you said, this is probably as good as it gets without leaving HTML/CSS.

File: paper-radio-button.html [hunk 1]
```diff
@@ -95,26 +95,23 @@
...
+      #offRadio, #onRadio {
         position: absolute;
         box-sizing: border-box;
         top: 0;
         left: 0;
-        right: 0;
         width: 100%;
         height: 100%;
         border-radius: 50%;
-        border: solid 2px;
```
Comment (bicknellr) [inline paper-radio-button.html:107]: Hmm... weird. I don't think this is / was ever valid.

**Comment by bicknellr** [comment]
LGTM, thanks!

**Pull request merged**